src/components/flex-container/flex-container.scss

```diff
@@ -49,3 +46,14 @@
 :host(limel-flex-container[justify='space-evenly']) {
     justify-content: space-evenly;
 }
+
+// Since consumers often copy/paste from examples of `limel-dialog` from the
+// documentations, they often put their buttons in a `limel-flex-container`
+// which is used in the footer slot of those examples with buttons.
+// So to prevent unwanted layout problems of buttons having no distance, we use
+// below code to target `limel-flex-container`s in those slots.
+:host(limel-flex-container[slot='button']) {
+    gap: 0.5rem; // Makes sure buttons get some default distance
+    width: 100%;
+    justify-content: flex-end;
+}
```

Comment:
Hmm… does this really have to be added in `flex-container.scss`? Since it's added on `:host`, I think it could be added by the dialog's css, and I wonder if that wouldn't make more sense?

Also, I've been thinking about whether we really need to specify that it should be limel-flex-container at all… 🤔 Maybe we could just say in the dialog css, that whatever element has the `slot="button"` attribute should have `display: flex` and these additional styles?

Reply:
Hmm… no… it doesn't seem like it can be added from the dialog's styling… 😞

But I did find another solution, which I've added as a separate commit.

Reply:
I really like your solution. Thanks for tweaking my commits!